### PR TITLE
[CRC] add docstring for some Variables

### PR DESCRIPTION
**Pull request by Difers**
add docstring for some Variables
#186

### CODE REVIEW

File: sot/opcode_translator/executor/variables/base.py [hunk 1]
```diff
@@ -138,6 +182,14 @@ class VariableBase:
     """
     VariableBase is a basic concept and each symbols in VM stack is regarded as
     an Variable Object in symblic tracing process.
+
+    It is designed to warp the variables traced by tracker.
```
Comment (NotHaozi) [inline sot/opcode_translator/executor/variables/base.py]: wrap

**Review by NotHaozi — COMMENTED** [review]
LGTM

File: sot/opcode_translator/executor/variables/base.py (hunk 1 above)
Comment (Difers) [reply sot/opcode_translator/executor/variables/base.py]: done~

**Review by NotHaozi — APPROVED** [review]

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Pull request merged**